tabpy-server/server_tests/test_service_info_handler.py

```diff
@@ -120,3 +120,66 @@ def test_given_tabpy_server_with_auth_expect_correct_info_response(self):
                 'required': True,
             }
         }, features)
+
+
+class TestServiceInfoHandlerWithoutAuth(AsyncHTTPTestCase):
+    @classmethod
+    def setUpClass(cls):
+        prefix = '__TestServiceInfoHandlerWithoutAuth_'
+
+        # create state.ini dir and file
+        cls.state_dir = tempfile.mkdtemp(prefix=prefix)
+        cls.state_file = open(os.path.join(cls.state_dir, 'state.ini'), 'w+')
+        cls.state_file.write('[Service Info]\n'
+                             'Name = TabPy Serve\n'
+                             'Description = \n'
+                             'Creation Time = 0\n'
+                             'Access-Control-Allow-Origin = \n'
+                             'Access-Control-Allow-Headers = \n'
+                             'Access-Control-Allow-Methods = \n'
+                             '\n'
+                             '[Query Objects Service Versions]\n'
+                             '\n'
+                             '[Query Objects Docstrings]\n'
+                             '\n'
+                             '[Meta]\n'
+                             'Revision Number = 1\n')
+        cls.state_file.close()
+
+        # create config file
+        cls.config_file = tempfile.NamedTemporaryFile(
+            prefix=prefix, suffix='.conf', delete=False)
+        cls.config_file.write(
+            bytes(
+                '[TabPy]\n'
+                'TABPY_STATE_PATH = {}'.format(
+                    cls.state_dir),
+            'utf-8'))
+        cls.config_file.close()
+        cls.config_file.close()
```

Comment:
Not sure if this is intentional but I think calling close twice should raise an error

Reply:
fixed